src/app/pages/Settings.tsx

```diff
@@ -4,16 +4,19 @@ import { Link } from "lib/woozie";
 import PageLayout from "app/layouts/PageLayout";
 import RevealSecret from "app/templates/RevealSecret";
 import DAppSettings from "app/templates/DAppSettings";
+import CustomNetworksSettings from "app/templates/CustomNetworksSettings";
 import ActivateAccount from "app/templates/ActivateAccount";
 import RemoveAccount from "app/templates/RemoveAccount";
 import About from "app/templates/About";
+import { ReactComponent as NetworkIcon } from "app/icons/network.svg";
 import { ReactComponent as SettingsIcon } from "app/icons/settings.svg";
 import { ReactComponent as KeyIcon } from "app/icons/key.svg";
 import { ReactComponent as StickerIcon } from "app/icons/sticker.svg";
 import { ReactComponent as OkIcon } from "app/icons/ok.svg";
 import { ReactComponent as MinusIcon } from "app/icons/minus.svg";
 import { ReactComponent as ExtensionIcon } from "app/icons/extension.svg";
 import { ReactComponent as AppsIcon } from "app/icons/apps.svg";
+import clsx from "clsx";
```

Comment:
Pls, stick to order of imports:
1) External(from NPM) modules;
2) Modules from "lib/**" dir;
3) Etc.